src/database/DatabaseManager.js

```diff
@@ -344,60 +421,183 @@ export class DataBaseManager {
             return false;
         }
         // Save all the responses related to the question
-        for (let i = 0; i < question.answers.length; ++i) {
-            let savedState = await this.saveResponse(i, question.answers[i].answer, question.id, idDiscussion);
+        for (let i = 0; i < aQuestion.answers.length; ++i) {
+            let savedState = await this.saveResponse(i, aQuestion.answers[i].answer, aQuestion.id, aIdDiscussion);
             if (!savedState) {
                 return false;
             }
         }
         return saved;
     }
 
+    /**
+     * Save a question of an admin in the database
+     * @param aQuestion object Question that represents the Question to save
+     * @param aIdDiscussion integer that is the id of the Discussion related to the question
+     * @param aUsername String that is the username of the admin
+     * @returns {Promise<boolean>} true if the save went well false otherwise
+     */
+    async saveQuestionAdmin(aQuestion, aIdDiscussion, aUsername){
+        let saved = await this.saveQuestion(aQuestion,aIdDiscussion);
+        if(!saved){
+            return false;
+        }
+        // Save the question as an admin Question in the database
+        let idAdmin = await this.getAdminId(aUsername);
+        if(idAdmin == null){
+            logger.debug(`Error when looking for username id`);
+            return false;
+        }
+
+        const questionAdminSave = new QuestionAdmin({
+            id: {
+                refQuestion: aQuestion.id,
+                refDiscussion: aIdDiscussion
+            },
+            administrator: idAdmin
+        });
+
+        await questionAdminSave.save()
+            .then(questionAdminSave => logger.debug(`Question of admin saved ${questionAdminSave}`))
+            .catch(err => {
+                logger.debug(`Error when saving Question of admin id = ${aQuestion.id}`);
+                console.log(err);
+                saved = false;
+            });
+        // If the save went wrong we exit the function and return false
+        if(!saved){
+            return false;
+        }
+        return saved;
```

Comment:
No need of the if statement here.
```suggestion
        return saved;
```